internal/oidcserver/oidcserver.go

```diff
@@ -163,6 +163,16 @@ func (o *oidcServer) UserInfo(w http.ResponseWriter, r *http.Request) {
 	// Do nothing
 }
 
+// Healthz ...
+func (o *oidcServer) Healthz(w http.ResponseWriter, r *http.Request) {
```

Comment:
Why is this part of oidcserver ? And why do we need a function ?

Reply:
For kubernetes ingress, it makes periodic calls to a server endpoint `/healthz` and expects a 200 response.  This is so that it can add multiple servers into a round-robin server group and to know when to remove a failed server from that group.